src/utils/print.js

```diff
@@ -0,0 +1,18 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+export const resultPrint = (getCarsInfo) => {
+  const entries = Object.entries(getCarsInfo());
+  const result = entries.map(([carName, advanceCount], idx) => {
+    const [_, name] = carName.split("-");
```

Comment:
`getWinners`에서는 이렇게 자동차 이름을 구하셨던데 통일하시면 더 가독성이 좋아질 것 같습니당!

```suggestion
    const name = carName.split("-")[1];
```

Reply:
아 그렇군요! 저는 스프레드 방식으로 의도를 명확히 하는게 더 좋다고 생각해서 `getWinners`를 수정 해야겠네요 감사합니다!